src/libcollections/fmt.rs

```diff
@@ -475,13 +475,21 @@
 
 #![stable(feature = "rust1", since = "1.0.0")]
 
+#[stable(feature = "rust1", since = "1.0.0")]
 pub use core::fmt::{Formatter, Result, Write, rt};
```

Comment:
This is a case where `rt` is unstable but it's part of a stable reexport, would it be possible to issue a warning and/or error about this?